src/utils.js

```diff
@@ -73,3 +73,23 @@ export function getUsers() {
     return data.users;
   });
 }
+
+export function getTopics() {
+  return axios
+    .get("https://nc-news-api-iv8w.onrender.com/api/topics")
+    .then(({ data }) => {
+      return data.topics.map((topic) => {
+        return topic.slug;
+      });
+    });
+}
+
+export function selectArticlesByTopic(allArticles, currentTopic) {
```

Comment:
Hey Pablo, Articles should not be filtered locally at the front end. They should be fetched from the API with the relevant topic using 'axios params'. For this, see NC Notes/Axios/Axios Params.